daisy_workflows/image_import/enterprise_linux/translate_el.sh

```diff
@@ -15,131 +15,13 @@
 
 set -x
 
+# Download translate_el.py
```

Comment:
Technically, we could just do all of this in the python script and nix the bash script entirely- in that case we would just set the startup script to the python script. Otherwise, I think this look good.

Reply:
I use the bash script to install python-guestfs and python-pycurl before running the python script
Do you have any suggestion in how we can do this without the bash script? So we can get rid of it?

Reply:
Oh. I see the problem. Yeah ok, we'll keep the bash script because we need the python dependencies installed. Never mind my comment :)